satpy/composites/__init__.py

```diff
@@ -668,11 +668,15 @@ def __init__(self, name, lim_low=85., lim_high=88., day_night="day_night", **kwa
             day_night (string): "day_night" means both day and night portions will be kept
                                 "day_only" means only day portion will be kept
                                 "night_only" means only night portion will be kept
+            need_alpha (bool): This only affects "day only" or "night only" result
+                               True means an alpha band will be added to the output image for transparency
+                               False means the output is just a single-band image with masked-out area
```

Comment:
I'm not sure how I feel about the word "need" here. I think to go along with other similar keyword arguments in Satpy, could we call this `include_alpha`?

```suggestion
            include_alpha (bool): This only affects the "day only" or "night only" result.
                               True means an alpha band will be added to the output image for transparency.
                               False means the output is a single-band image with undesired pixels being masked out (replaced with NaNs).
```

The above change reflects my other suggestion in my review comment.